app/main.py

```diff
@@ -1 +1,19 @@
-# write your code here
+def copy_file(command: str) -> None:
+    parts = command.split(" ")
+
+    if len(parts) < 3 or parts[0] != "cp":
+        return
+
+    source_path = parts[1]
+    destination_path = parts[2]
+
+    if source_path == destination_path:
+        return
+
+    with open(source_path, "r") as file_in, \
+         open(destination_path, "w") as file_out:
```

Comment:
try to avoid using backslash  when you move to new line

Reply:
Check if the first part of the command is "cp"